tracer/src/Datadog.Trace/Ci/Ipc/IChannel.cs

```diff
@@ -0,0 +1,18 @@
+// <copyright file="IChannel.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System;
+
+namespace Datadog.Trace.Ci.Ipc;
+
+internal interface IChannel : IDisposable
+{
+    int BufferBodySize { get; }
```

Comment:
Well header size is a constant that's why I didn't expose it

Reply:
So is `BufferBodySize` 😛 But if it's not required by external users, then agree there's no point exposing. If, on the other hand, anything external _is_ assuming the size of the header, then I think we should expose it instead (in case it changes in the future, for example)